src/main/java/com/iota/iri/service/dto/wereAddressesSpentFrom.java

```diff
@@ -1,5 +1,9 @@
 package com.iota.iri.service.dto;
 
+/**
+ * This class represents the core API request 'wereAddressesSpentFrom'.
+ * Check if a list of addresses was ever spent from, in the current epoch, or in previous epochs.
+ **/
```

Comment:
check whether all the `*Response` classes docs violate DRY

Reply:
still violates DRY. You have the same line in API.java.
I see in the mdx it only appears once